### ISSUE TITLE
any plans to support https://github.com/Famous/engine/tree/v1.0.0?

### ISSUE THREAD

**Issue by quentinburley** [IjzerenHein/famous-flex#108]
I note that famous-0.3.5 has been deprecated in favour of famous-1.0.0.

any plans to support https://github.com/Famous/engine/tree/v1.0.0?

Thanks

**Comment by IjzerenHein** [comment]
Hi, good question. The mixed mode API is all new and different and also the new famous-platform library has not yet been released. I'll make a short announcement on this topic probably next week.
cheers.

**Comment by dcsan** [comment]
after talking to a couple people there:

- no more famous scrollviews: "just use native scrolling"
new browsers don't stop scrolling when JS is running, so it makes no sense to try to do pure JS scrolls.

- no 'appmode' flag anymore. 
not sure if we will get scrollevents on our own surfaces for `overflow: scroll` got mixed responses on this question. 

- no modifiers
you can just add transforms to surfaces.

- separated into BEST which is app framework and CORE for the core drawing engine.
BEST is a more declarative syntax for Behaviors etc. but said it won't ship for at least 4~5 months (in famous time, that could be a year?). Zach from angularfamous got hired to lead that project, and it looks like a mix of react and angular with DI and $state

- they rewrote the engine from scratch
since they had no tests before and lots of turnover of the team

- no text support for webGL
just meshes and images.

**Comment by IjzerenHein** [comment]
Awesome post @dcsan, thanks for sharing that here!

**Comment by bardu** [comment]
Any news on possible support for famous-1.0.0?

**Comment by IjzerenHein** [comment]
Hi, no official statements yet, but here is what I wrote to someone else earlier this week:

So I think that mixed mode is a big step forward. I like the API a lot better and it addresses several important things such as garbage collection and being able to define differential sizes easily, and that's a big thing. At this stage only the engine has been released, but if you want to build apps, I think we really need the new "famous platform" as well (which should contain a scrollview control), which will be available as a public beta next month.
So, I've been playing around with the new API and trying to get a good feel for it. I have a basic version of a LayoutController up & running and I will be releasing that in the near future. I'm also working on AutoLayout (cassowary constraints) as I myself have a big need to make doing layout even easier and faster.
Concerning upgrading my famous projects. Some libs that I have created try to solve a particular problem with famo.us (e.g. famous-bkimagesurface, famous-sizeconstraint). These things can now be achieved properly with famo.us itsself, so I don't intend to update those. Others, like famous-map and famous-flex, I will continue to support. My goal is to update these projects in a way so that they have an API that matches the new vanilla famo.us / framework API. I think that's important and that developers only have to learn one style of API. Because of this I'm waiting in anticipation of the famous framework and BEST. When released I will probably update my projects shortly after that. In the meantime I will be working on getting the internals ported and probably add some GL features.
Hope that gives you a glimpse of what's coming next.
